app/views/styles/component/_form_footer.scss

```diff
@@ -145,6 +145,62 @@
         &__main-controls {
             margin-bottom: 0;
             margin-top: 0;
+
+            .next-prompt-container {
+                width: 100%;
+                margin-bottom: 10px;
+                text-align: center;
+
+                input[type=checkbox] {
```

Comment:
To get the same checkbox styling as in the form, you can simply change the HTML syntax a bit. See the removed code in _form-footer.pug in this commit: https://github.com/enketo/enketo-express/commit/15b29116023123acf8d4781b5f3d2d1298ed7340 (we should have a tickbox for save as draft). I believe the `<label class="question"><input type="checkbox"><span class="option-label">some text</span></label>` structure will do the trick. CSS rules are here (in enketo-core): https://github.com/enketo/enketo-core/blob/dbc18f2cb382e58af98c90138b443ac2407dd61a/src/sass/formhub/_radiocheck.scss#L26

Reply:
got it, code updated